app.js

```diff
@@ -49,10 +49,47 @@ app.use(express.json());
 // logging middleware
 app.use(morgan("tiny"));
 
+app.post("/api/upload-json-metadata-to-ipfs", async (req, res) => {
```

Comment:
@manangouhari  Just need a tiny update. Please wrap it in try & catch block and please implement a proper edge case for handling error.

Reply:
Handled here - #7